docs/organizations/what-are-organizations.md

```diff
@@ -23,6 +23,8 @@ This opens the list of organizations on your Git providers. The organization wit
 
 ## Updates on the Git provider
 
+<!-- TODO PLUTO-368 Review this section -->
```

Comment:
@jorgebraz I need the team's help to clarify how **changes to the repositories** on the Git provider will be reflected on Codacy with the new behavior.
Currently, some changes are automatically reflected, such as renaming a repository, changing the visibility, or deleting it.
With the new behavior, how will this work? Will these changes be reflected in a different way? Won't be reflected at all?

Can you (or someone else on the team) help me to update this table? Thanks!

![image](https://user-images.githubusercontent.com/26439949/227311983-9d328b4c-ebe9-468e-b4ed-bfbc8cf34b3a.png)

Reply:
@franciscoovazevedo @lolgab Maybe you can also help me with this content. 🙏 😉

Reply:
@machadoit Can you please jump in to help us with these details? 🙏 

Namely, when renaming/changing visibility/deleting a repository on the Git provider, will Codacy use the webhooks events to update the information on the local/follow tables?
Or will the local repo information be now out of date regarding these Git provider updates?

Reply:
> when renaming/changing visibility/deleting a repository on the Git provider, will Codacy use the webhooks events to update the information on the local/follow tables?

It does not update the follow tables, as there is nothing to update there on those cases* (deletes are removed automatically with DB cascades). It does update the ProjectTable with hooks and whatnot as it did before the feature. So everything should still show as expected.

The tracking on the follow table, is based on (userId, repoId) pair, meaning that any change to the user itself, or the repo itself shouldn't matter, as the tracking is made using ids. (Example: the rename of a repo is not affected by follow, as it retains same database identifier)

That said, the important cases that matter are related to the user losing access to the repo, like 'remove member from organization or group'. Some time ago, during a @franciscoovazevedo I mentioned that the we should also 'unfollow' when a member is removed from the organization, not sure if he created a task or did such work, maybe he can confirm. Regardless, worst case scenario, a repository may not disappear from the list, but when the use clicks it, it will not be able to access it and it will be auto-unfollowed.

Something important things to keep in mind:

* The intention is to NOT affect those cases. (if we do, we should improve/fix it)
* We don't rely on follow table information alone, it takes into consideration the repository information stored that does not depend on the follow feature. The follow information is only used for listing, so anything related to the information of a repo, or actions/access to the repo itself are not affected by the follow feature

Reply:
Thanks, @machadoit! 👍 
Thus, it seems we don't need any updates to this part of the documentation, as the repository information will keep being synchronized the same way as before.